Introduction_hyp3_mintpy.ipynb

```diff
@@ -0,0 +1,500 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction to time-series analysis of HyP3 product with MintPy"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Introduction\n",
+    "This notebook shows you how to do time-series analysis against HyP3 product with MintPy. This tutorial assumes that you have already successfully installed the MintPy and created the conda environment.\n",
+    "Detail installation info is refferered at https://github.com/insarlab/MintPy."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Run the application"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Before running the program, some package need to be installed and imported."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!conda install -y -c conda-forge hyp3_sdk"
```

Comment:
I'd recommend installing `ipywidgets` here as well.  `hyp3_sdk.watch()` requires `ipywidgets` in a jupyter environment, but it's not included by default, leading to an error in the `watch()` cell later in the notebook.

![Screenshot from 2021-06-16 13-58-53](https://user-images.githubusercontent.com/17994518/122300385-b963ff80-ceab-11eb-92ef-2de066355d63.png)